CorrelatorSharp/packages.config

```diff
@@ -1,4 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
.nuspec is used to create the .nupkg and packages.config is used at build/nuget restore time to download the required dependencies from nuget.org. When you install a package in a project in Visual Studio - it adds it to the packages.config. So, if I check out your branch - I won't be able to build it now, because it won't download and install the Immutable collections package.